requirements/common.txt

```diff
@@ -191,3 +191,7 @@ git+https://github.com/rafaelmartins/pyoembed.git@eb9901917c2a44b49e2887c077ead8
 
 # Needed to run tests in parallel
 tblib==1.3.0
+
+# Needed for gitlint
+arrow==0.10.0
+sh==1.11
```

Comment:
these should go in `dev.txt` with what they're needed for, right?  

BTW @umairwaheed `tblib` should be in `dev.txt` too, right?